Sources/Data+String.swift

```diff
@@ -0,0 +1,23 @@
+// Copyright © 2017-2018 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import Foundation
+
+extension Data {
+    public func toString() -> String {
```

Comment:
I don't think these extensions are useful. They don't provide any extra clarity on use and introduce an unexpected side effect of retiring empty string on error.

Reply:
Yes they are redundant, will remove them, cheers.